admin-dev/themes/default/template/controllers/modules_positions/form.tpl

```diff
@@ -120,5 +121,9 @@
 			$(this).change(position_exception_listchange);
 		});
 	});
+	$('select[name=id_hook]').on('change',function ()
+	{
```

Comment:
Can you remove the line break and add space after the comma?
Are you sure about binding event on the element outside the `$(document).ready()`?